src/styles/_variables.scss

```diff
@@ -30,3 +30,11 @@ $dark-text: $dark-rock;
 $warning: $yellow;
 $error: $red;
 $accent: $medium-green;
+
+$logo: #282828;
+
+// fonts
+
+$logo-font: 'Arbutus Slab', 'Sans-Serif';
+
+$arbutus: 'Arbutus', 'Sans-Serif';
```

Comment:
Make this a semantic variable name.

Follow established patterns for fallback font. https://developer.mozilla.org/en-US/docs/Web/CSS/font-family